### PR TITLE
Switch to using new png fonts + .osd file format v2 support:

### PR DESCRIPTION
**Pull request by benlumley**
- Use PNG font files instead of the old bin format
- Only 2 fonts now; one for sd + one for hd, rather than separate pages
- But the PNG can have equivalent of 4 pages; so BF 4.5 colour is supported
- OSD v2 support; which uses a 4 char string (BTFL/INAV etc) as the FC identifier. Can still handle v1 files too.
- Font files now optional; it will use the FC variant to load defaults from the msp-osd repo.

### CODE REVIEW

**Comment by benlumley** [comment]
(corresponds with upcoming msp-osd 0.12.x)

File: src/osd-overlay/osd.ts [hunk 1]
```diff
@@ -7,13 +7,30 @@ interface OsdHeader {
...
+
+interface OsdConfigV1 {
   charWidth: number;
   charHeight: number;
   fontWidth: number;
   fontHeight: number;
   xOffset: number;
   yOffset: number;
   fontVariant: number;
+
```
Comment (stylesuxx) [inline src/osd-overlay/osd.ts:33]: Unnecessary empty line.

**Review by stylesuxx — CHANGES_REQUESTED** [review]
Just one nit picky comment, otherwise everything looks fine to me.

**Review by stylesuxx — APPROVED** [review]

**Comment by benlumley** [comment]
thanks @stylesuxx

**Pull request merged**